### PR TITLE
Add Simplified Chinese translation for kubectl

### PR DESCRIPTION
**Pull request by shiywang**
What this PR does / why we need it:
This PR provides first attempt to translate kubectl in Simplified Chinese.

Which issue this PR fixes (optional, in fixes #<issue number>(, fixes #<issue_number>, ...) format, will close that issue when PR gets merged): fixes #
No issues

Special notes for your reviewer:
Although I'm a native speaker for Mandarin Chinese, but I think translation is a whole different knowledge which I'm not good at it, so this pr absolutely need to be polished.
@adohe @mengqiy @resouer @k82cn @caesarxuchao @wanghaoran1988 sorry I think there are so many folks who are good at Chinese I haven't mention, feel free to leave a comment on it : )
also cc @brendandburns

### CODE REVIEW

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.8\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: zh\n"
+
+# https://github.com/kubernetes/kubernetes/blob/masterpkg/kubectl/cmd/apply.go#L98
+msgid "Apply a configuration to a resource by filename or stdin"
+msgstr "通过文件或者标准输入的读取方式apply一个资源"
```
Comment (vfreex) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 通过文件名或标准输入流(stdin)对资源进行配置。

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_cluster.go#L38
+msgid "Delete the specified cluster from the kubeconfig"
+msgstr "从kubeconfig文件中删除指定的集群"
```
Comment (vfreex) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: “删除 kubeconfig 文件中指定的集群”
中英文混排时用空格隔开。

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/delete_context.go#L38
+msgid "Delete the specified context from the kubeconfig"
+msgstr "从kubeconfig文件中删除指定的上下文"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_contexts.go#L62
+msgid "Describe one or many contexts"
+msgstr "描述一个或更多上下文"
```
Comment (vfreex) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 更多->多个

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_clusters.go#L40
+msgid "Display clusters defined in the kubeconfig"
+msgstr "从kubeconfig配置文件中显示指定的集群"
```
Comment (vfreex) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 显示 kubeconfig 文件中指定的集群

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+msgid "Update the annotations on a resource"
+msgstr "更新资源的注解"
+
+msgid ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgid_plural ""
+"watch is only supported on individual resources and resource collections - "
+"%d resources were found"
+msgstr[0] "watch 仅在单独的一个资源或者一个资源集合上被支持 -资源 %d 被发现"
```
Comment (vfreex) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 建议改为主动语态，如
watch 仅支持单独的资源或者资源集合 - 找到了 %d 个资源

**Review by vfreex — CHANGES_REQUESTED** [review]

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po (hunk 4 above)
Comment (vabc3) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 显示kubeconfig配置文件中定义的集群

Comment (shiywang) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: specified cluster -> 指定的集群
clusters defined -> 定义的集群 ?

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+msgid "Describe one or many contexts"
+msgstr "描述一个或更多上下文"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/get_clusters.go#L40
+msgid "Display clusters defined in the kubeconfig"
+msgstr "从kubeconfig配置文件中显示指定的集群"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/view.go#L64
+msgid "Display merged kubeconfig settings or a specified kubeconfig file"
+msgstr "显示合并的集群或者指定的kubeconfig配置文件"
```
Comment (vabc3) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: 合并的kubeconfig配置？

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+msgid "Sets an individual value in a kubeconfig file"
+msgstr "设置 kubeconfig 文件中的一个单独值"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/use_context.go#L48
+msgid "Sets the current-context in a kubeconfig file"
+msgstr "设置 kubeconfig 文件中的当前上下文"
+
+# https://github.com/kubernetes/kubernetes/blob/master/pkg/kubectl/cmd/config/unset.go#L47
+msgid "Unsets an individual value in a kubeconfig file"
+msgstr "取消设置 kubeconfig 文件中的一个单独值"
```
Comment (rootfs) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: `单独值` sounds odd, anything better?

**Comment by pwittrock** [comment]
I was actually joking when I said I could review this  :P 

Looks like this doesn't impact non-chinese aspects of kubectl, so I can approve once @rootfs lgtm's

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po (hunk 7 above)
Comment (shiywang) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: updated, how about now ?

File: translations/kubectl/zh_CN/LC_MESSAGES/k8s.po [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"X-Generator: Poedit 1.8.8\n"
+"X-Poedit-SourceCharset: UTF-8\n"
+"Language-Team: \n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+"Language: zh\n"
+
+# https://github.com/kubernetes/kubernetes/blob/masterpkg/kubectl/cmd/apply.go#L98
```
Comment (shiywang) [inline translations/kubectl/zh_CN/LC_MESSAGES/k8s.po]: updated

**Comment by mengqiy** [comment]
LGTM
@rootfs Any more comments?

**Comment by rootfs** [comment]
great work @shiywang @mengqiy

**Comment by rootfs** [comment]
/release-note-action-required
Add Simplified Chinese translation for kubectl

**Comment by mengqiy** [comment]
@pwittrock ping for approval

**Comment by brendandburns** [comment]
@k8s-bot pull-kubernetes-e2e-gce-etcd3 test this
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by shiywang** [comment]
@k8s-bot pull-kubernetes-federation-e2e-gce test this

**Comment by shiywang** [comment]
@k8s-bot pull-kubernetes-e2e-kops-aws test this

**Comment by CaoShuFeng** [comment]
I would like invate you to review this change:
https://github.com/kubernetes/kubernetes/pull/46487

**Pull request merged**